t/t5325-commit-graph-bloom.sh

```diff
@@ -0,0 +1,255 @@
+#!/bin/sh
```

Comment:
This file still doesn't have the executable bit set. [Use these instructions](https://stackoverflow.com/a/21694391/127088).